services/workers/src/main/resources/table-cluster-sqs-spb.xml

```diff
@@ -42,12 +42,12 @@
 					</bean>
 				</property>
 				<property name="targetMethod" value="run" />
-				<property name="concurrent" value="false" />
+				<property name="concurrent" value="true" />
 			</bean>
 		</property>
 		<!-- We stager the start delay of each trigger to spread out the timing -->
 		<property name="startDelay" value="256" />
```

Comment:
With such a long repeatInterval we need to increase the start delay (since it might misfire the first time it starts up) to avoid having longer tests.